core/store/src/main/java/mil/nga/giat/geowave/core/store/adapter/statistics/StatisticsProvider.java

```diff
@@ -22,4 +22,6 @@
 
 	public EntryVisibilityHandler<T> getVisibilityHandler(
 			ByteArrayId statisticsId );
+
+	public DataAdapter<T> getDataAdapter();
```

Comment:
getDataAdapter is not called anywhere from what I can tell, not sure we need it on the interface?